package.json

```diff
@@ -58,6 +58,7 @@
     "rar",
     "gz",
     "bz2",
+    "ps",
```

Comment:
I know this is crazy, but can you put it at the end so everything is in the same order?